packages/dashboard/src/components/CompositeList/CompositeList.styles.ts

```diff
@@ -0,0 +1,128 @@
+import { IButtonStyles } from 'office-ui-fabric-react/lib/Button';
+import { ICommandBarStyles } from 'office-ui-fabric-react/lib/CommandBar';
+import { ISearchBoxStyles } from 'office-ui-fabric-react/lib/SearchBox';
+
+import { DefaultFontStyles, DefaultPalette, IStyle } from 'office-ui-fabric-react/lib/Styling';
+
+export const getCommandBarStyle = (): ICommandBarStyles => {
+  return {
+    root: {
+      backgroundColor: 'white',
```

Comment:
>backgroundColor: 'white', [](start = 6, length = 25)

use pallate

Reply:
DefaultPalette.white

---
In reply to: [249970039](https://github.com/OfficeDev/office-ui-fabric-react/pull/7739#discussion_r249970039) [](ancestors = 249970039)